c++/test/tcompound.cpp

```diff
@@ -115,7 +115,7 @@ test_compound_2()
             s_ptr->d    = i * 8 + 6;
             s_ptr->e    = i * 8 + 7;
         }
-        memcpy(buf, orig, nelmts * sizeof(src_typ_t));
+        HDmemcpy(buf, orig, nelmts * sizeof(src_typ_t));
 
```

Comment:
I plan to eventually remove everything private and then the HD prefixes will go away, but until then I'll keep the pedantry.